src/functions.c

```diff
@@ -808,7 +808,7 @@ void functionFlushCommand(client *c) {
 
 void functionHelpCommand(client *c) {
     const char *help[] = {
-"LOAD <ENGINE NAME> <LIBRARY NAME> [REPLACE] [DESC <LIBRARY DESCRIPTION>] <LIBRARY CODE>",
+"LOAD <ENGINE NAME> <LIBRARY NAME> [REPLACE] [DESCRIPTION <LIBRARY DESCRIPTION>] <LIBRARY CODE>",
```

Comment:
the json file also needs a change.
and maybe some examples in redis-doc?

Reply:
Yes, just update the json file as well.

Reply:
@oranagra the docs already have examples with description on the introduction page: https://redis.io/topics/functions-intro#expanding-the-library